packages/frontend/web-ui/src/common/helpers/getUserMeId.ts

```diff
@@ -0,0 +1,15 @@
+import { buildSerializableResponse } from '../http/helpers/buildSerializableResponse';
+import { UserMeResponse } from '../http/models/UserMeResponse';
+import { UserMeSerializedResponse } from '../http/models/UserMeSerializedResponse';
+
+import { httpClient } from '../http/services/HttpService';
+
+export const getUserMeId = async (
+  token: string | null,
```

Comment:
why is this field nullable

Reply:
I've changed this. I supposed that as the variable could be null, I extended this type in my code.